pkg/prometheus/agent/operator.go

```diff
@@ -759,6 +759,9 @@ func createSSetInputHash(p monitoringv1alpha1.PrometheusAgent, c operator.Config
 		http2 = p.Spec.Web.WebConfigFileFields.HTTPConfig.HTTP2
 	}
 
+	// Ignoring changes to RevisionHistoryLimit field
```

Comment:
Sorry for being dense in my previous comment: can you update the comment here too?

Reply:
No worries